src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/weapons/SwordOfBeheading.java

```diff
@@ -86,9 +94,9 @@ public EntityKillHandler getItemHandler() {
 
                         e.getDrops().add(skull);
                     }
-                    break;
-                default:
-                    break;
+                }
+                default -> {
+                }
```

Comment:
I believe you can remove this entirely right? Or just inline it if not

Reply:
biscuit wanted me to keep default in one of my previous PR, so i'll just inline it.